client/src/pages/selectMusicians.tsx

```diff
@@ -46,7 +38,26 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-export default function ChoosePlayers() {
+export default function SelectMusicians() {
```

Comment:
Naming of this page is vexing... thoughts? SelectMusiciansAndInstruments seems too long.